### DISCUSSION TITLE
is there MAUI support planned?

### DISCUSSION THREAD

**Discussion by gsgou** [crowdin/xamarin-sdk#8] [General]
can we expect a MAUI sdk any time soon?

**Comment by andrii-bodnar** [comment]
There was an attempt to add MAUI support - https://github.com/crowdin/xamarin-sdk/pull/5

Anyway, you can always utilize the Crowdin CDN endpoints described here - https://support.crowdin.com/content-delivery/#for-web-applications

(Originally answered here https://github.com/crowdin/xamarin-sdk/issues/7#issuecomment-2022880999)
